### ISSUE TITLE
Loading activity data from the database

### ISSUE THREAD

**Issue by pbastia** [bcgov/cas-reporting#345]
The purpose of this ticket is to implement loading of activity data from the database. Currently there is no code to do that, we'll need a service that takes in (report_version, activity_id) and builds a json_data object from the records in:

-   report_activity
-   report_source_type
-   report_unit
-   report_fuel
-   report_emission
-   report_methodology

The data will fetch the activity data, add a `source_type` property with the source type data, etc. etc.

#### Acceptance Criteria:

Given I am an industrial reporter
When I edit an activity page on my report
Then the data saved in the database is loaded into the form

#### Development Checklist:

- [x] a service that fetches the json data into the above mentioned tables and puts it all together in a single json_data object
- [x] an API endpoint `reporting/load-activity` that takes in: report_version_id, facility_id, activity_id
- [x] The activity page fetches that data upon loading
- [x] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By
#332 
#367 

#### Notes:

-

**Comment by pbastia** [comment]
@patriciarussellCAS cc for visibility, @dleard and I are adding this ticket as a high priority item. This will need to be done right after #332

**Comment by pbastia** [comment]
This will allow you to check #382 @patriciarussellCAS
